webapp/channels/src/packages/mattermost-redux/src/constants/preferences.ts

```diff
@@ -92,9 +92,9 @@ const Preferences = {
             sidebarTextHoverBg: '#28427b',
             sidebarTextActiveBorder: '#5d89ea',
             sidebarTextActiveColor: '#ffffff',
-            sidebarHeaderBg: '#192a4d',
+            sidebarHeaderBg: '#192A4D',
```

Comment:
Nitpicking, but it's weird to me that the existing RGB values are lower case and the new ones are upper case. Mind making those consistent and updating the snapshots?

Reply:
Updated in the latest commit